rbx_dom_weak/src/instance.rs

```diff
@@ -33,11 +33,13 @@ let dom = WeakDom::new(data_model);
 */
 #[derive(Debug)]
 pub struct InstanceBuilder {
-    pub(crate) referent: Ref,
     pub(crate) name: String,
     pub(crate) class: String,
     pub(crate) properties: HashMap<String, Variant>,
     pub(crate) children: Vec<InstanceBuilder>,
+
+    /// The referent of the instance that this `InstanceBuilder` refers to.
+    pub referent: Ref,
```

Comment:
I don't think we can just make this field public, it would not be allowed to mutate it to any value as this would violate a lot of assumptions. We can add a getter instead, named `referent()`.